build/yaml/templates/dotnet-package-steps.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 3.1.x'
+  inputs:
+    version: 3.1.x
+
+- task: DotNetCoreCLI@2
+  displayName: 'Run `dotnet restore`'
+  inputs:
+    command: restore
+    projects: '$(WorkingDirectory)\*.csproj'
+    feedsToUse: config
+    nugetConfigPath: packages\NuGet.Config
+
+- task: DotNetCoreCLI@2
+  displayName: 'Run `dotnet build`'
+  inputs:
+    projects: '$(WorkingDirectory)\*.csproj'
+    arguments: '--configuration $(BuildConfiguration) --no-restore'
+    workingDirectory: '$(WorkingDirectory)'
+
+- task: CopyFiles@2
```

Comment:
Should this not use dotnet publish?

Reply:
I was following conventions from the starter template but if dotnet publish does the same I'll replace with that.